go/engine/kex2_provisionee.go

```diff
@@ -597,6 +614,33 @@ func (e *Kex2Provisionee) cacheKeys() (err error) {
 	return nil
 }
 
+func (e *Kex2Provisionee) SigningKey() (libkb.GenericKey, error) {
+	if e.eddsa == nil {
+		return nil, fmt.Errorf("provisionee missing signing key")
```

Comment:
Yeah, I get what it does, I just wonder why it's better to in general use both interfaces instead of just the `fmt` one that always works. Just curious, I'll change it anyway.